persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_13/generated/entities/generated_client.bal

```diff
@@ -4,56 +4,226 @@
 // It should not be modified by hand.
 
 import ballerina/persist;
+import ballerina/sql;
+import ballerinax/mysql;
+import ballerina/time;
```

Comment:
order the import in alphabetical order

Reply:
updated in [aa95e9c](https://github.com/ballerina-platform/persist-tools/pull/106/commits/aa95e9c0a62f306236f4dc03ae1d6f60593a1b6e)